docs/feature_unicode.md

```diff
@@ -4,11 +4,11 @@ There are three Unicode keymap definition methods available in QMK:
 
 ## `UNICODE_ENABLE`
```

Comment:
Code tags in headers end up looking strange with the current Docsify theme. Maybe this could be improved. I like the idea of having sort of standard sections for feature pages, these would go under "Usage". See again the Backlight docs.

Reply:
Yeah, I've noticed this as well. Will change, thanks!

Reply:
That’s a common pattern already, and lately I’ve been thinking about another round of documentation site improvements. I’d rather see us make the theme not look weird. :)

Reply:
That works too. Should I leave it as is for now?